src/commands/junit/upload.ts

```diff
@@ -251,14 +251,16 @@ export class UploadJUnitXMLCommand extends Command {
   }
 
   private async getMatchingJUnitXMLFiles(spanTags: SpanTags): Promise<Payload[]> {
-    const jUnitXMLFiles = (this.basePaths || []).reduce((acc: string[], basePath: string) => {
-      const isFile = !!path.extname(basePath)
-      if (isFile) {
-        return acc.concat(fs.existsSync(basePath) ? [basePath] : [])
-      }
+    const jUnitXMLFiles = (this.basePaths || [])
+      .reduce((acc: string[], basePath: string) => {
+        const isBasePathFile = isFile(basePath)
+        if (isBasePathFile) {
```

Comment:
Is there a reason ```if (isFile(basePath))``` isn't applicable here?

Reply:
did that change in https://github.com/DataDog/datadog-ci/pull/946/commits/37d1d26afdfff06550c417a0de21ea0a886414f7